pkg/test-infra/tidb/operation.go

```diff
@@ -63,8 +63,8 @@ type Ops struct {
 }
 
 // New ...
-func New(namespace, name string, config fixture.TiDBClusterConfig) *Ops {
-	return &Ops{cli: tests.TestClient.Cli, tc: RecommendedTiDBCluster(namespace, name, config),
+func New(namespace, name string, config fixture.TiDBClusterConfig, enableTiFlash bool) *Ops {
```

Comment:
How about putting enableTiFlash into fixture.TiDBClusterConfig and using TiFlashReplicas to control whether or not require tiflash?

Reply:
Yes, that is a good idea